crates/swift-bridge-ir/src/codegen/generate_rust_tokens/shared_enum.rs

```diff
@@ -140,7 +140,8 @@ impl SwiftBridgeModule {
         // User derives
         let mut derive_impl_ffi_bridges = vec![];
 
-        if shared_enum.derive.debug {
+        // Current derive(Debug) implementation could cause memory safety issues: https://github.com/chinedufn/swift-bridge/pull/194#discussion_r1134386788
```

Comment:
Minor tweak to make it clear that we're preventing the memory safety issue.

```rust
// We currently only allow derive(Debug) on non data carrying enums since
// to prevent a potential memory safety issue.
// https://github.com/chinedufn/swift-bridge/pull/194#discussion_r1134386788
```